pkg/network/msg.go

```diff
@@ -59,3 +59,15 @@ type DataRequest struct {
 	ID        string `json:"id"`
 	SensorIds []int  `json:"sensorIds"`
 }
+
+// DataPublish represents the incoming publish data command
+type DataPublish struct {
+	ID   string `json:"id"`
+	Data []Data `json:"data"`
+}
+
+// Data represents the thing's data
+type Data struct {
```

Comment:
The same comments I did on this [PR](https://github.com/CESARBR/knot-babeltower/pull/50) applied here.

Reply:
Done!